sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/online_deployment.py

```diff
@@ -662,41 +659,42 @@ def __init__(
-        :param model: Model entity for the endpoint deployment, defaults to None
+        :keyword model: Model entity for the endpoint deployment, defaults to None
         :type model: typing.Optional[typing.Union[str, ~azure.ai.ml.entities.Model]]
-        :param code_configuration: Code Configuration, defaults to None
+        :keyword code_configuration: Code Configuration, defaults to None
         :type code_configuration: typing.Optional[CodeConfiguration]
-        :param environment: Environment entity for the endpoint deployment, defaults to None
+        :keyword environment: Environment entity for the endpoint deployment, defaults to None
         :type environment: typing.Optional[typing.Union[str, ~azure.ai.ml.entities.Environment]]
-        :param app_insights_enabled: Is appinsights enabled, defaults to False
+        :keyword app_insights_enabled: Is appinsights enabled, defaults to False
         :type app_insights_enabled: bool
-        :param scale_settings: How the online deployment will scale, defaults to None
+        :keyword scale_settings: How the online deployment will scale, defaults to None
         :type scale_settings: typing.Optional[typing.Union[DefaultScaleSettings, TargetUtilizationScaleSettings]]
-        :param request_settings: Online Request Settings, defaults to None
+        :keyword request_settings: Online Request Settings, defaults to None
         :type request_settings: typing.Optional[OnlineRequestSettings]
-        :param liveness_probe: Liveness probe settings, defaults to None
+        :keyword liveness_probe: Liveness probe settings, defaults to None
         :type liveness_probe: typing.Optional[ProbeSettings]
-        :param readiness_probe: Readiness probe settings, defaults to None
+        :keyword readiness_probe: Readiness probe settings, defaults to None
         :type readiness_probe: typing.Optional[ProbeSettings]
-        :param environment_variables: Environment variables that will be set in deployment, defaults to None
+        :keyword environment_variables: Environment variables that will be set in deployment, defaults to None
         :type environment_variables: typing.Optional[typing.Dict[str, str]]
-        :param instance_type: _description_, defaults to None
+        :keyword instance_type: Azure compute sku, defaults to None
         :type instance_type: typing.Optional[str]
-        :param instance_count: _description_, defaults to None
+        :keyword instance_count: The instance count used for this deployment, defaults to None
         :type instance_count: typing.Optional[int]
-        :param egress_public_network_access: Whether to restrict communication between a deployment and the
+        :keyword egress_public_network_access: Whether to restrict communication between a deployment and the
              Azure resources used to by the deployment. Allowed values are: "enabled", "disabled", defaults to None
-        :param code_path: _description_, defaults to None
```

Comment:
Was deleting this line intended? It seems like the type and comment below corresponds to code_path. Maybe you wanted to replace param by keyword?